### PR TITLE
Add Flipnote Lenny exploit setup page for users without Unlaunch

### LINKED ISSUE #90: Download all tools needed before launching the exploit
Suggestion: If you ask the user to download dumpTool and Unlaunch to their SD card before they launch the exploit, they can just follow along instead of turning off the system, swapping the SD card over to their computer and then back to the DSi, etc.
Not sure if this has been brought up before.

### PR DESCRIPTION
**Pull request by RocketRobz**
This PR makes Flipnote Lenny a recommended exploit, mainly because:
1. In Memory Pit, there a part of RAM past the first 4MB which is write-protected (and has no known fix).
2. Compatibility with DSi mode titles and homebrew is improved.
3. It is the best available exploit for an Unlaunch-free experience.

Even though Memory Pit is accessible to all DSi consoles with a working camera (and can be used to install Unlaunch), it is slightly bugged, which is why this PR make Flipnote Lenny a more recommended exploit than Memory Pit.
Of course, it is up to the user to chose either.

I suppose it would look better if the exploit tabs we're made collapsible or something, so that the exploit page looks cleaner, but idk how to do that, if at all possible.

This PR also removes unneeded steps for configuring TWiLight Menu++.

This PR closes #90.

EDIT: This PR instead now adds a new exploit setup page for Flipnote Lenny instead of adding it to the main page, which should make it less confusing.

### CODE REVIEW

**Comment by Epicpkmn11** [comment]
Personally I'm not a fan of this change because:

1. Memory Pit is both easier to use and more compatible
2. Adding a second main exploit to the guide would greatly increase confusion with new users

IMO better solutions would be, in order of my preference:

- Mention on either Launching the Exploit or maybe Installing Unlauch that if you don't want to install Unlaunch then using Flipnote Lenny will get you better compatibility in certain ways, at the cost of taking more effort to load the exploit and link to Alternate Exploits
- Swap Memory Pit and Flipnote Lenny, so Lenny is the main exploit the guide recommends (not a big fan of this option either)
- This but with tabs and Memory Pit as the recommendation

There are some other improvements that could be made, for example Alternate Exploits could be renamed to Launching the Exploit (Flipnote Lenny) since that's the only alternate. I just think this hurts the ease of use more than it helps.

Other opinions very welcome.

**Comment by RocketRobz** [comment]
I meant Flipnote Lenny. (See edited first post.)

File: docs/launching-the-flipnote-exploit.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+Launching the exploit will boot you into TWiLight Menu++, a homebrew application that acts as a replacement to the DSi Menu.
+
+## Requirements
+- A way to copy downloaded files to your SD card
+- An application that can extract archives, such as [7-Zip](https://www.7-zip.org/) (Windows) or [The Unarchiver](https://apps.apple.com/us/app/the-unarchiver/id425424353) (macOS)
+   - We advise you to not use WinRAR, as it is known to break things
+- Flipnote Studio installed on your Nintendo DSi
+   - If you do not have Flipnote Studio, you may still be able to redownload it from the Nintendo DSi Shop, if you have downloaded it previously
+   - If you are unable to redownload it, then go to the previous page for Memory Pit instead
```
Comment (Epicpkmn11) [inline docs/launching-the-flipnote-exploit.md:15]:
```suggestion
   - If you do not have Flipnote Studio, then go to the previous page and use Memory Pit instead
```
The shop is dead, no use checking it.

File: docs/launching-the-exploit.md [hunk 2]
```diff
@@ -2,17 +2,15 @@
 title: Launching the Exploit
 ---
 
-We'll start by first setting up your SD card to launch our exploit. For most users, we recommend using an exploit called "Memory Pit" which takes advantage of a flaw in how the Nintendo DSi Camera application handles image metadata.
+We'll start by first setting up your SD card to launch our exploit. For most users, we recommend using an exploit called "Memory Pit" which takes advantage of a flaw in how the Nintendo DSi Camera application handles image metadata.
```
Comment (Epicpkmn11) [inline docs/launching-the-exploit.md:5]:
```suggestion
We'll start by first setting up your SD card to launch our exploit. For most users, we recommend using an exploit called "Memory Pit" which takes advantage of a flaw in how the Nintendo DSi Camera application handles image metadata.     

```
I think you need a blank line for it to split properly

File: docs/launching-the-exploit.md [hunk 3]
```diff
@@ -2,17 +2,15 @@
 title: Launching the Exploit
 ---
 
-We'll start by first setting up your SD card to launch our exploit. For most users, we recommend using an exploit called "Memory Pit" which takes advantage of a flaw in how the Nintendo DSi Camera application handles image metadata.
+We'll start by first setting up your SD card to launch our exploit. For most users, we recommend using an exploit called "Memory Pit" which takes advantage of a flaw in how the Nintendo DSi Camera application handles image metadata.     
+If you don't plan on installing Unlaunch, it is recommended to use [Flipnote Lenny](lauching-the-flipnote-exploit) instead. ([Why?](faq.html#why-use-flipnote-lenny-over-memory-pit))
```
Comment (Epicpkmn11) [inline docs/launching-the-exploit.md:6]: Might be good to either link the unlaunch benefits here or change that FAQ to "Which is the best exploit for me?" or so with the benefits of all three.

**Review by Epicpkmn11 — COMMENTED** [review]
Looks better imo, though a couple minor things that might be worth changing:

Also curious if NightScript, hansol, and/or emiyl have any thoughts before merging.

**Comment by lifehackerhansol** [comment]
I don't particularly see why we can't make Flipnote the main exploit. It's not _that_ hard. 

We even have the zoogie certified:tm: visual guide if we need to https://zoogie.github.io/web/flipnote_directions/ (made for frogminer but close enough)

**Comment by Epicpkmn11** [comment]
there's still a few more chances for user error and it takes longer and more effort. It's also not all that uncommon for a DSi to not have Flipnote and that only gets worse now that the shop is dead any time someone decides to delete it or format their DSi.

Memory Pit is overall the better exploit unless your camera's broken or you run into issues with certain homebrew so I think the current state of this PR is pretty good besides maybe expanding on that FAQ to include Unlaunch. I might try something for that in the morning.

**Comment by lifehackerhansol** [comment]
> It's also not all that uncommon for a DSi to not have Flipnote

imo this is where I'd link to the Memory Pit guide. 

The problem I see with mentioning Unlaunch right off the bat is the structure of the guide in general. We don't ever describe _what_ Unlaunch is and _why_ one would install it except on the `Installing Unlaunch` page where it is relevant. The initial purpose of the PR makes much more sense in that regard instead of telling people to do this with terminology that isn't really easily explainable even in FAQ.

**Comment by Epicpkmn11** [comment]
I just did some changes that I think should make it a bit more clear:
- Moved the "what will this guide do" to the bottom of the home page and rewrote it to explain the different exploit options, with a link to the FAQ explaining in more detail
   - This way we aren't asking people to make a choice before explaining what they're choosing
- Changed the FAQ to a single FAQ pros/cons list of all three main options
- Added tabs to the Unlaunch cleanup section for Memory Pit/Lenny

The current state of the PR is live at https://dsi.ピケ.コム, feedback on my latest commit is very welcome.

**Comment by RocketRobz** [comment]
I think it looks good to me.

**Comment by lifehackerhansol** [comment]
[0001-Add-a-Get-Started-page.zip](https://github.com/cfw-guide/dsi.cfw.guide/files/8468741/0001-Add-a-Get-Started-page.zip)

This `.patch` file adds a `Get Started` page, debloating the homepage a bit, and allows to go a bit further in-depth on each exploit and why one should use it. Since now we have two exploit paths, it would be better to have a landing page for which exploit to choose.

Other guides feature a page like this: 
- https://wii.guide/get-started.html
- https://3ds.hacks.guide/get-started.html

**Comment by lifehackerhansol** [comment]
[0001-Move-TWiLight-Menu-and-dumpTool-download-steps-to-ge.zip](https://github.com/cfw-guide/dsi.cfw.guide/files/8469048/0001-Move-TWiLight-Menu-and-dumpTool-download-steps-to-ge.zip)

This patch further extends on the previous patch, moving TWiLight Menu++ and dumpTool download steps to the new get-started page, as discussed over on Discord.

**Review by Epicpkmn11 — APPROVED** [review]
I think this looks good to me now, unless someone else still has any suggestions.

File: docs/launching-the-exploit.md [hunk 4]
```diff
@@ -85,22 +60,12 @@ If you enter the SD card camera album and nothing unusual happens, ensure that y
...
-   - The default region setting, Game-specific, is generally recommended
-   - After setting them, if the console crashes on white screens, press and hold POWER to power off, power back on, and re-run the exploit
-1. Once you're in TWiLight Menu++, press SELECT to switch to the DS Classic Menu
-1. Tap the DS icon in the bottom middle to open settings
-1. Use the <kbd class="l">L</kbd> / <kbd class="r">R</kbd> or <kbd class="face">X</kbd> / <kbd class="face">Y</kbd> buttons to switch over to the "Misc. settings" page
-1. Switch the "SysNAND Region" entry to your console's region
-1. Press <kbd class="face">B</kbd> to save and exit TWiLight Menu++ settings
+You should now see the TWiLight Menu++ language selection and should select the language and region you prefer to use. They do not need to match your console's language or region.
 
 You are now able to use TWiLight Menu++, however it's recommended to continue on and make a NAND backup. This can potentially be used to save your console if anything bad happens in the future.
```
Comment (lifehackerhansol) [inline docs/launching-the-exploit.md:65]: I believe this line was always there, but now it feels janky considering the new line we just added above.

**Review by lifehackerhansol — APPROVED** [review]

File: docs/troubleshooting.md [hunk 5]
```diff
@@ -17,7 +17,7 @@ If the NAND was saved, follow [Restoring NAND](restoring-nand) and continue with
 
 ### There is no audio or boot splash when launching "Launcher" using Unlaunch
 
-The developer of Unlaunch (nocash) has intentionally patched out the background audio and boot-splash by default. You can regain these effects by [reinstalling Unlaunch](installing-unlaunch) using TWiLight Menu++, or by using [hiyaCFW](https://wiki.ds-homebrew.com/hiyacfw/installing).
+The developer of Unlaunch (nocash) has intentionally patched out the background audio and boot-splash by default. You can regain these effects by [reinstalling Unlaunch](installing-unlaunch.html) using TWiLight Menu++, or by using [hiyaCFW](https://wiki.ds-homebrew.com/hiyacfw/installing).
```
Comment (NightScript370) [inline docs/troubleshooting.md:20]: You should mention that it would have to be with the patches toggle.

**Review by NightScript370 — APPROVED** [review]
When I initially written the guide, this was moreso meant to be a "do it your way" guide.
I'm glad to see this further enhances said goals. Kudos to all those involved, because you guys are the real heroes.

**Pull request merged**